website/docs/guides/1.0-upgrade-guide.html.markdown

```diff
@@ -46,13 +46,49 @@ provider "azurestack" {
 
 At a high level, version 1.0 to includes the following changes:
 
+* [The `enpoint` property was deprecated in favour of the `metadata_hostname` property](#the-endpoint-property-has-been-replaced-with-metadata_hostname).
 * [Changes when importing existing resources](#changes-when-importing-existing-resources) - when bringing Resources provisioned outside of Terraform under Terraform's control, these resources will now be required to be imported.
 * [Custom Timeouts will be available on Resources](#custom-timeouts-for-resources) - this will allow you to specify a custom timeout for provisioning the resource in your Terraform Configuration [using the `timeouts` block](https://www.terraform.io/docs/configuration/resources.html#operation-timeouts).
 * [New resources for Virtual Machines and Virtual Machine Scale Sets](#new-resources-for-virtual-machines-and-virtual-machine-scale-sets).
 * [Removing Fields, Data Sources and Resources which have been deprecated](#removal-of-deprecated-fields-data-sources-and-resources).
 
+
 Each of these topics is covered in more detail below.
 
+## The `endpoint` property has been replaced with `metadata_hostname`
+
+The be more descriptive and better align with the AzureRM provider and the provider block property `arm_endpoint` has been deprecated in favour of the `metadata_hostname` property which expects a FQDN hostname instead of a URI. For instance if you were setting it in the provider block previously:
```

Comment:
```suggestion
The be more descriptive and better align with the AzureRM provider and the provider block property `arm_endpoint` has been deprecated in favour of the `metadata_host` property which expects a FQDN hostname instead of a URI. For instance if you were setting it in the provider block previously:
```